examples/README.md

```diff
@@ -684,6 +684,13 @@ For this example, the following is needed in the selected region:
 
 Creates a Slurm cluster with [Parallelstore] instance mounted.
 
+To provision the cluster, please run:
+
+```text
+./gcluster create examples/ps-slurm.yaml --vars "project_id=${GOOGLE_CLOUD_PROJECT}"
```

Comment:
Why not use a single command syntax `./gcluster deploy examples/ps-slurm.yaml --vars "project_id=${GOOGLE_CLOUD_PROJECT}"`